test/core/elem.wast

```diff
@@ -161,3 +161,54 @@
   )
   "unknown table 0"
 )
+
+;; Test element sections
+(module $module-1
+  (type $out-i32 (func (result i32)))
+  (table 10 anyfunc)
+  (elem (i32.const 8) $const-i32-a)
+  (elem (i32.const 9) $const-i32-b)
+  (func $const-i32-a (type $out-i32) (i32.const 65))
+  (func $const-i32-b (type $out-i32) (i32.const 66))
+  (func (export "call-7") (type $out-i32)
+    (call_indirect $out-i32 (i32.const 7))
+  )
+  (func (export "call-8") (type $out-i32)
+    (call_indirect $out-i32 (i32.const 8))
+  )
+  (func (export "call-9") (type $out-i32)
+    (call_indirect $out-i32 (i32.const 9))
+  )
+  (export "shared-table" (table 0))
+)
+
+(register "test")
```

Comment:
Nit: `(register "test" $module-1)` for extra clarity. Also, maybe give it a more descriptive name than "test", or call it "module-1"..